roles/httpd/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: install httpd
+  yum:
+      name: httpd
```

Comment:
not sure if we should really follow that pattern. The package name won't change over a release. Trying to fit for multiple destributions might be a tough target anyways. config files and versions differ anyways.